recipe/meta.yaml

```diff
@@ -37,13 +29,13 @@ outputs:
         - make
         - pkg-config
       host:
-        - abseil-cpp
+        - libabseil
         - libprotobuf
-        - zlib 1.2.11
+        - zlib
         # coin or deps
         - coin-or-utils
         - coin-or-osi
-        - coin-or-clp =1.17.4
+        - coin-or-clp
```

Comment:
This was added because it is the only version that was supported as newer versions had a bug, if I remember correctly. As pointed out, they don't get propagated to run though, so this might need to get fixed.

@tkralphs Sorry to be pinging you of the blue but do you know if everything is resolved with or-tools now and we can remove this?

Reply:
> As pointed out, they don't get propagated to run though, so this might need to get fixed.

What is getting propagated is "1.17" not "1.17.4"
https://github.com/conda-forge/coin-or-clp-feedstock/pull/5/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR18

Reply:
Yes which is probably a mistake but I cannot remember fully what the situation was. I would prefer to wait for Ted and then merge but I can also ask or-tools otherwise.

Reply:
Understood

Reply:
> As pointed out, they don't get propagated to run though, so this might need to get fixed.

You can add `coin-or-clp !=1.17.5,!=1.17.6 to the run requirements (though builds against 1.17.7 would automatically get that lower bound from the run export)

Reply:
@tkralphs Gentle ping if you have a second.

Reply:
Merging this now since we didn't hear from upstream. I guess we can always patch the metadata if we get issues.